packages/redux/src/promotionEvaluations/types/action.types.ts

```diff
@@ -0,0 +1,30 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type { Error } from '@farfetch/blackout-client/types';
+import type {
+  PromotionEvaluationId,
+  PromotionEvaluationItem,
+} from '@farfetch/blackout-client/promotionEvaluations/types';
+
+export interface FetchPromotionEvaluationItemsRequestAction extends Action {
+  meta: { promotionEvaluationId: PromotionEvaluationId };
+  type: typeof actionTypes.FETCH_PROMOTION_EVALUATION_ITEMS_REQUEST;
+}
+export interface FetchPromotionEvaluationItemsSuccessAction extends Action {
+  meta: { promotionEvaluationId: PromotionEvaluationId };
+  type: typeof actionTypes.FETCH_PROMOTION_EVALUATION_ITEMS_SUCCESS;
+  payload: {
+    result: Array<PromotionEvaluationItem>;
+  };
+}
+export interface FetchPromotionEvaluationItemsFailureAction extends Action {
+  meta: { promotionEvaluationId: PromotionEvaluationId };
+  type: typeof actionTypes.FETCH_PROMOTION_EVALUATION_ITEMS_FAILURE;
+  payload: { error: Error | unknown };
```

Comment:
Is this `unknown` really necessary? 🤔

Reply:
I added it because of an eslint warning at `fetchPromotionEvaluationItemsFactory.ts` but the same error seems to be present in all factories, so I removed it.